README.md

```diff
@@ -33,8 +33,8 @@ ncspot is available on macOS (Homebrew), Windows (Scoop), Linux (native package
 BSD's. Detailed installation instructions for each platform can be found [here](/doc/users.md).
 
 ## Configuration
-A configuration file can be provided at `$XDG_CONFIG_HOME/ncspot/config.toml`. Detailed
-configuration information can be found [here](/doc/users.md#configuration).
+A configuration file can be provided. Detailed configuration information can be found
```

Comment:
Yes, I agree it's good to provide information that should be true for most people. I just don't know whether `~/.config/` really says much, as it is only true for Linux and the BSD's, and only if the user doesn't use XDG Base Directories to change the location. Maybe just mention the platform's usual configuration directory instead of `~/.config`?

Reply:
Sorry, yeah, that's what I meant (`~/.config/ncspot`)